agent/bench-scripts/postprocess/uperf-postprocess

```diff
@@ -68,7 +64,7 @@ my @cpu_busy;	# each array element contains a hash with
 		# hostname: hostname or IP
 		# role: client, server, host, kvm-host, container-host, etc.
 		# timeseries: a hash of timestamp:value key-pairs
-my %efficiency;
+# my %efficiency;
```

Comment:
Delete this line?

Reply:
See [subsequent comment](https://github.com/distributed-system-analysis/pbench/pull/2349#discussion_r716930094).